e2e-tests/portal/cypress/e2e/journeys/admin/dashboard/dashboard-table-data.spec.js

```diff
@@ -27,7 +27,7 @@ context('Admin dashboard', () => {
       deal = insertedDeal;
     });
 
-    cy.insertOneGefApplication(GEF_DEAL_DRAFT, BANK1_MAKER1).then((dealGef) => {
```

Comment:
Please ensure applications are inserted as `BANK1_MAKER1` as an `ADMIN` will never create an application in a real scenario.